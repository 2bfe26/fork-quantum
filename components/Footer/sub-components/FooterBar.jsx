import React from 'react';
import styled from 'styled-components';
import { Container, Row, Col } from '../../Grid';

const CathoFooterInfo = styled.div`
  font-family: ${props => props.theme.fontFamily};

  font-size: ${props => props.theme.fontSize};
  padding:20px 0px 15px 0px;
  margin-left:20px;
  a {
    color: ${props => props.theme.primaryColor};
    text-decoration: none;
    margin-left:5px;
    @media (max-width: 980px) {
      color: #CCC;
      text-decoration: underline;
    }
  }
`;

const FooterBarContainer = styled(Container)`{
  background-color: #e8e8e8;
  width: ${props => (props.fluid ? 'auto' : props.theme.containerWidth)};
  color: ${props => props.theme.secondaryColor};
  @media (max-width: 980px) {
    background-color:#333;
    color: #CCC;
  }
}`;

const ColFlex = styled(Col)`{
  display: flex;
  align-items: center;
}`;

const Catho = () => (
  <CathoFooterInfo>
    Catho Online Ltda. CNPJ: 03.753.088/0001-00 Alameda Juari, 262a - Barueri, SP - 06460-090
    <a target="_blank" rel="noopener noreferrer" href="https://www.catho.com.br/institucional/privacidade-relacionamento/" >
         Pol√≠tica de Privacidade
    </a>
  </CathoFooterInfo>
);

export default () => (
  <FooterBarContainer fluid>
    <Row>
      <Col desktop={12}>
        <FooterBarContainer>
          <Row>
            <ColFlex desktop={12}>
              <img alt="Catho logo" src={`${process.env.DOMAIN}/ms/fragments-footer/static/cathoFooter.png`} />
              <Catho />
            </ColFlex>
          </Row>
        </FooterBarContainer>
      </Col>
    </Row>
  </FooterBarContainer>
);
