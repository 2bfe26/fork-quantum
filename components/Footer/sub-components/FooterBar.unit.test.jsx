import React from 'react';
import renderer from 'react-test-renderer';
import FooterBar from './FooterBar';

describe('Footer component', () => {
  it('Should match the snapshot', () => {
    const tree = renderer.create(<FooterBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
