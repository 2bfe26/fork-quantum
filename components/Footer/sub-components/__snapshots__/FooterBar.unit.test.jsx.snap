// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should match the snapshot 1`] = `
<div
  className="FooterBar__FooterBarContainer-s17byu91-1 jrtWCQ Container-v8sf5n-0 fmnGah"
>
  <div
    className="Row-s1166su-0 gYiQGp"
  >
    <div
      className="Col-bi16rv-0 cygcwq"
    >
      <div
        className="FooterBar__FooterBarContainer-s17byu91-1 lcQbr Container-v8sf5n-0 gUDuIj"
      >
        <div
          className="Row-s1166su-0 gYiQGp"
        >
          <div
            className="FooterBar__ColFlex-s17byu91-2 guzMgG Col-bi16rv-0 cygcwq"
          >
            <img
              alt="Catho logo"
              src="/ms/fragments-footer/static/cathoFooter.png"
            />
            <div
              className="FooterBar__CathoFooterInfo-s17byu91-0 fRtWfd"
            >
              Catho Online Ltda. CNPJ: 03.753.088/0001-00 Alameda Juari, 262a - Barueri, SP - 06460-090
              <a
                href="https://www.catho.com.br/institucional/privacidade-relacionamento/"
                rel="noopener noreferrer"
                target="_blank"
              >
                Pol√≠tica de Privacidade
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
